packages/sdk/src/sdk/services/DiscoveryNodeSelector/healthChecks.test.ts

```diff
@@ -150,7 +154,7 @@ describe('health_check', () => {
       expect(reason).toBe('slot diff')
     })
 
-    // test('comms unhealthy', () => {
+    // it('comms unhealthy', () => {
```

Comment:
This is curious.... changed from test->it but still commented? 😄

Reply:
decided against re-enabling this in this pr, might do anohter though :)